src/FormBuilder/resources/js/form-builder/src/components/settings/ControlForPopover/index.tsx

```diff
@@ -1,7 +1,7 @@
 import type {ReactNode} from 'react';
 import {BaseControl, Button} from '@wordpress/components';
-import {edit as EditIcon} from '@wordpress/icons';
 import classNames from 'classnames';
+import PopoverEditIcon from './PopoverEdicIcon';
```

Comment:
small typo (will need to update file name too)

```suggestion
import PopoverEditIcon from './PopoverEditIcon';
```

Reply:
Thanks! Sorry my eyes glazed right over that. f8e9549721c0671b52928ec4213d316074a7f17e